addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```
@@ -21,6 +21,11 @@
 
 .o_ChatterTopbar_tools {
     padding-right: var(--ChatterTopbar-padding-right, 0);
+    border-bottom: var(--ChatterTopbar-border, $border-width solid $border-color);
```

Does this work? Shouldn't be ?
```suggestion
    border-bottom: var(--ChatterTopbar-border, #{$border-width solid $border-color});
```

---

Please correct me if I'm wrong , but is it necessarry since it's SCSS variables inside? I think they will be compiled as static CSS value and therefore do not need interpolation.

---

After testings and as you mentioned it's necessary when used in variable definition but works in normal usage, to avoid any weird behaviour it's better to use it anyway.